helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -22,102 +22,75 @@
 {{ template "chart.description" . }}
 <!-- markdownlint-enable line-length -->
 
```

Comment:
...it may not need to be before the TOC, but it feels like we should still reference those items..

Reply:
They're in there still, but would a warning section near the top be better to pull more attention to it?

Reply:
I don't honestly know if folks go to the top of this page, or really go to this page ever at all...

"if it were me" I would probably put it in the "upgrading from" section (and since it's me, I would probably put it in each of the sections, assuming that folks only read the one that pertains to them)... Of course, this means going back and figuring out when orchestrators got introduced and only adding notes about it to the appropriate sections...